dev/nextjs-project/data/default.json

```diff
@@ -1 +1,7 @@
\ No newline at end of file
+{
```

Comment:
I think this file should be deleted.

In VSCode it's there an option to quickly discard changes you rather not commit.

![image](https://user-images.githubusercontent.com/370612/121236603-d0e52c00-c89e-11eb-9db3-62a6edf16ab0.png)

Reply:
I'm trying to resolve this, but wouldn't `git rm --cached` delete the file from remote too? I'm looking into how I might undo the diffs..

Reply:
I think so. Do you see any side effect of removing the file?

Reply:
The user would have to create the same page to pass build.test.js.